pkg/api/server/v1alpha2/auth/rbac.go

```diff
@@ -89,11 +109,32 @@ func (r *RBAC) Check(ctx context.Context, namespace, resource, verb string) erro
 			continue
 		}
 
+		user := tr.Status.User.Username
+		UID := tr.Status.User.UID
+		groups := []string{"tekton.dev"}
+		extra := map[string]authzv1.ExtraValue{}
+
+		// Check whether the authenticated user has access to impersonate
```

Comment:
Replace "access" with permission. Admins grant _access_, users receive _permission_.

```suggestion
		// Check whether the authenticated user has permission to impersonate
```

Reply:
:thumbsup: Changed.